src/components/plots/areaPlot.ts

```diff
@@ -141,6 +168,22 @@ export module Plots {
 
       return { data: allPlotData.data, pixelPoints: allPlotData.pixelPoints, selection: d3.selectAll(allElements) };
     }
+
+    protected _constructAreaProjector(xProjector: _Projector, yProjector: _Projector, y0Projector: _Projector) {
+      var definedProjector = (d: any, i: number, dataset: Dataset) => {
+        var positionX = Plot._scaledAccessor(this.x())(d, i, dataset);
+        var positionY = Plot._scaledAccessor(this.y())(d, i, dataset);
+        return Utils.Methods.isValidNumber(positionX) && Utils.Methods.isValidNumber(positionY);
+      };
+      return (datum: any[], index: number, dataset: Dataset) => {
```

Comment:
Now the naming of the argument seems a bit weird, since it should be something like "`arrayDatum`"...